integration_tests/header.sh

```diff
@@ -14,6 +14,21 @@ function finish {
 }
 trap finish EXIT
 
+function assert_run_outcome_is {
```

Comment:
I'd leave a basic function comment here, explaining what the function accepts as parameters and what it does; it's mostly self-documenting, but an extra comment line wouldn't hurt :)